tests/PyzTest/Yves/Checkout/Controller/CheckoutControllerTest.php

```diff
@@ -101,8 +101,8 @@ protected function setUp(): void
      */
     protected function skipIfCi()
```

Comment:
Tests works fine with GA?

Reply:
Workflow is green

Reply:
We don't use CirleCI anymore, we can remove it.